src/main/java/uk/gov/hmcts/cmc/claimstore/idam/models/UserDetails.java

```diff
@@ -21,4 +33,18 @@ public Long getId() {
     public String getEmail() {
         return email;
     }
+
+    public String getForename() {
+        return forename;
+    }
+
+    public Optional<String> getSurname() {
+        return Optional.ofNullable(surname);
+    }
+
+    @JsonIgnore
+    public String getFullName() {
+        return getSurname().map(surname -> String.join(" ", forename, surname))
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Rename "surname" which hides the field declared at line 16. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.reform.hmcts.net/coding_rules#rule_key=squid%3AHiddenFieldCheck)